dkc/core/models/user_quota.py

```diff
@@ -1,26 +1,29 @@
 from typing import Type
 
+from django.apps import apps
 from django.contrib.auth.models import User
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django_girders.models import SelectRelatedManager
 
 
-class Quota(models.Model):
-    allocation = models.BigIntegerField(default=0)
-    user = models.OneToOneField(User, null=True, on_delete=models.CASCADE)
-    # root_set
+class UserQuota(models.Model):
+    user = models.OneToOneField(
+        User, primary_key=True, on_delete=models.CASCADE, related_name='quota'
```

Comment:
Note to reviewers, I'd like to propose this pattern as a good idea generally for one-to-one relationships like this. It removes the unnecessary surrogate key from this table and uses the target relation's pkey as a natural pkey instead.